index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Blessed Jason Mwanza   - Portfolio for  Microverse </title>
+  <link rel="stylesheet" href="css/index.css" type="text/css">
+</head>
+<body>
+  <header class="top-header d-flex">
+    <a href="#" class="header-title">Welcome</a>
+    <button class="mobile-nav-trigger">
+      <img src="images/Union.png" alt="">
+    </button>
+  </header>
+  <section id="about-me" class="d-flex">
+    <h1 class="d-flex">
+     <span class="self-center">Hey there. I’m Blessed</span>
+     <span class="green-text self-center">I’m a software developer</span>
+    </h1>
+    <p>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hesitate to contact me.
+    </p>
+    <div class="social-links self-center">
+      <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+      <a href="#"><img src="images/Linkedin-icon.svg" alt="LinkedIn"></a>
+      <a href="#"><img src="images/medium.svg" alt="Medium"></a>
+      <a href="#"><img src="images/github.svg" alt="GitHub"></a>
+      <a href="#"><img src="images/angellist.svg" alt="Angellist"></a>
```

Comment:
- [ ] [OPTIONAL] There is one last thing we can improve here, Kindly let's have these social media links inside a list since this should be a list of links, for this you can use the unordered list `<ul>`, see the example below 👇 
```
<ul>
   <li>
        <a href="your social media link goes here"><img src="./assets/img/twitter.svg" alt=""></a>
   </li>
   <li>
        <a href="your social media link goes here"><img src="./assets/img/Linkedin-icon.svg" alt=""></a>
   </li>
</ul>
```

This will keep our code more semantic 👍

Reply:
Thank you a lot 😃 . 

I am right on it! Of course, my energy levels have just been boosted to improve on the advised changes now.

>  HappyCoding

Reply:
That's wonderful, keep up the good work 🍾 👏